### DISCUSSION TITLE
Cày thuật toán cùng ông dev

### DISCUSSION THREAD

**Discussion by wtrungvu** [OngDev/community#17] [Video Request]
1. Cùng nhau thử thách phá đảo các câu hỏi phỏng vấn HackerRank, LeetCode,...
2. Giải thích với mọi người vì sao nên làm vậy? Làm thế nào để tối ưu?Giải thích dễ hiểu, kinh nghiệm phỏng vấn...
3. Cùng nhau tham khảo và học hỏi các thuật toán khác, lời giải khác tốt hơn...

**Comment by thangtv611** [comment]
Bài toán tính số Fibonacci kinh điển. Hãy implement tất cả các cách sao cho tối ưu nhất.

**Comment by cutbypham** [comment]
https://www.youtube.com/watch?v=qM02HKYgm4I&list=PLoaAbmGPgTSPkemjPThpTBljuTprLNBeR
